metricbeat/docs/gettingstarted.asciidoc

```diff
@@ -267,13 +280,17 @@ docker run {dockerimage}
 [source,shell]
 ----------------------------------------------------------------------
 sudo chown root metricbeat.yml <1>
-sudo ./metricbeat -e -c metricbeat.yml -d "publish"
+sudo chown root modules.d/system.yml <1>
+sudo ./metricbeat run -e -c metricbeat.yml -d "publish"
 ----------------------------------------------------------------------
 <1> You'll be running Metricbeat as root, so you need to change ownership
-of the configuration file (see
+of the configuration file and any configurations enabled in the `modules.d`
+directory (see
 {libbeat}/config-file-permissions.html[Config File Ownership and Permissions]
 in the _Beats Platform Reference_).
 
+//REVIEWERS: It seems really inconvenient to users to expect them to change ownership on all the files in `modules.d`. This is something I had to do, but not 100% sure if there is a better way. Please advise.
+
```

Comment:
I agree we may need to think about something here, we have `strict.perms` flag to avoid this (with the cost of losing some security). deb/rpm packages also ave these permissions right by default. Not sure we can help on the tar.gz case